save_test.go

```diff
@@ -298,6 +299,24 @@ func TestSaveByValue(t *testing.T) {
 	assert.Equal(t, ErrStructPtrNeeded, err)
 }
 
+func TestSaveWithBatch(t *testing.T) {
+	db, cleanup := createDB(t, Batch())
+	defer cleanup()
+
+	var wg sync.WaitGroup
+
+	for i := 0; i < 5; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			err := db.Save(&User{ID: i + 1, Name: "John"})
+			assert.NoError(t, err)
```

Comment:
Not that it matter, but I prefer to use `require.NoError` in cases where it does not make sense to continue testing.